src/Context/RestContext.php

```diff
@@ -98,7 +98,7 @@ public function theResponseShouldBeEmpty(): void
     {
         $actual = $this->request->getContent();
         $message = "The response of the current page is not empty, it is: $actual";
-        $this->assertTrue(null === $actual || '' === $actual, $message);
```

Comment:
`$actual` is always `string`.

https://github.com/minkphp/Mink/blob/b4ff07f7f9bd3158c52016b1309e3430f0739482/src/Element/DocumentElement.php#L33